doc/nrf/libraries/bluetooth_services/services/throughput.rst

```diff
@@ -17,26 +17,26 @@ The GATT Throughput Service is used in the :ref:`ble_throughput` sample.
 Service UUID
 ************
 
-The 128-bit service UUID is 0483DADD-6C9D-6CA9-5D41-03AD4FFF4ABB.
+The 128-bit service UUID is ``0483DADD-6C9D-6CA9-5D41-03AD4FFF4ABB``.
 
 Characteristics
 ***************
 
 This service has one characteristic.
 
-Throughput (0x1524)
-===================
+Throughput (``0x1524``)
+=======================
 
 Write Without Response
    * Write any data to the characteristic to measure throughput.
-   * Write 0 bytes to the characteristic to reset the metrics.
+   * Write zero bytes to the characteristic to reset the metrics.
```

Comment:
Hmm  it looks like in real it is one byte which values is not relevant, need to be written.
Maybe it is not related to this PR but it is worth to fix here